pom.xml

```diff
@@ -472,24 +472,6 @@
             <artifactId>jose4j</artifactId>
             <version>0.9.3</version>
         </dependency>
-        <dependency>
-            <groupId>org.jvnet.hudson.plugins</groupId>
-            <artifactId>favorite</artifactId>
-            <version>2.3.1</version>
-            <exclusions>
-                <!-- Upper bound fix: Remove it once token-micro plugin is fixed to remove dependency on asm-tree
-                     PR https://github.com/jenkinsci/token-macro-plugin/pull/31. Fix to be available in 2.4.0.
-                -->
-                <exclusion>
-                    <groupId>org.ow2.asm</groupId>
-                    <artifactId>asm</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.ow2.asm</groupId>
-                    <artifactId>asm-tree</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
```

Comment:
No need to pin such an old version in the dependency management section. The linked PR is five years old.